src/Emogrifier.php

```diff
@@ -1251,12 +1256,22 @@ private function getBodyElement(\DOMDocument $document)
      *
      * will be parsed into the following array:
      *
-     *   "css" => "h1 { color:red; }"
-     *   "media" => "@media { h1 {}}"
+     *   0 => [
+     *     "css" => " h1 { color:red; } ",
+     *     "media" => ""
+     *   ],
+     *   1 => [
+     *     "css" => " h1 {}",
+     *     "media" => "@media "
+     *   ],
+     *   2 => [
+     *     "css" => " "
```

Comment:
Will/should this really be a string that just contains a space? Should we trim this?

Reply:
I've added `trim` for the CSS outside of @media rules, and these parts are now only included if the result of `trim` is a nonempty string.